### ISSUE TITLE
Configuration required?

### ISSUE THREAD

**Issue by warren8r** [mhart/epipebomb#4]
Hello,
I am trying to make use of epipebomb to stifle an issue I am having with a sms service (Textbelt).
I have installed epipebomb - is there there a configuration I should make after installing, as it seems I am getting the same EPIPE error.
Thank you so much for your time and attention.
Sincerely,
Matt

**Comment by mhart** [comment]
@warren8r what stream are you getting the EPIPE error on?

This module is usually used when you're piping `process.stdout` – for the command line.

You can use it on other streams if you like – and you can override the default behaviour (which is just to exit silently) by passing in your own handler function:

```js
var epipebomb = require('epipebomb')

epipebomb(myStream, myOnPipeFunction)
```

**Comment by warren8r** [comment]
Hello mhart,
Thank you so much for your response.
I feel silly as I am not even invoking your service! I simply installed it.
Anyway, I am a newbie. I have been writing .NET web apps for years, but this is my first time hosting someone elses code on a Linux server...it has been fun experience thus far and the folks I have encountered have been INCREDIBLY patient and kind.
Let me try your recommendation.
I am not particularly familiar with the innards of Textbelt, but I am sure I can find this stream and invoke epipebomb on it.
I will surely keep you posted.
Thank you so much again!
Matt
